LrcParser.Tests/Parser/Lrc/Lines/LrcLyricParserTest.cs

```diff
@@ -39,16 +39,16 @@ public void TestDecode(string lyric, LrcLyric expected)
             {
                 Text = "帰り道は",
                 StartTimes = [17000],
-                TimeTags = TestCaseTagHelper.ParseTimeTags(["[0,start]:0", "[1,start]:1000", "[2,start]:2000", "[3,start]:3000", "[3,end]:4000"]),
+                TimeTags = TestCaseTagHelper.ParseTimeTags(["[0,start]:0", "[1,start]:1000", "[2,start]:2000", "[3,start]:3000", "[4,end]:4000"]),
             },
         ],
         [
-            "[00:17.00] 帰<00:01.00>り<00:02.00>道<00:03.00>は<00:04.00>",
+            "[00:01.00] 帰<00:01.10>り<00:02.00>道<00:03.00>は<00:04.00>",

```

Comment:
What is the result if not change the test case?

Reply:
Exactly, that's why I changed the test case. The previous version was still wrong from the time where relative times were assumed.

Reply:
I might keep this test case as an edge case.

It's ok to make the refactor, but not change the test case.
Prefer to make it to another PR and describe why have those changes if have breaking change because lrc format has no strict spec.
Which means it's hard to decide whether to allow those changes merge or not if it's mixed with the refactor change.